plugins/repository-s3/src/main/java/org/elasticsearch/repositories/s3/S3ClientSettings.java

```diff
@@ -81,11 +81,11 @@
 
     /** The number of retries to use when an s3 request fails. */
     static final Setting.AffixSetting<Integer> MAX_RETRIES_SETTING = Setting.affixKeySetting(PREFIX, "max_retries",
-        key -> Setting.intSetting(key, S3Repository.Repositories.MAX_RETRIES_SETTING, 0, Property.NodeScope));
+        key -> Setting.intSetting(key, 3, 0, Property.NodeScope));
```

Comment:
I wonder if we should default to `ClientConfiguration.DEFAULT_RETRY_POLICY.getMaxErrorRetry()` instead?
Or to `PredefinedRetryPolicies.DEFAULT_MAX_ERROR_RETRY`?